src/App.svelte

```diff
@@ -38,6 +38,7 @@
       ...(appId
         ? [{ url: `/apps/${app}/app_ids/${appId}/`, name: appId }]
         : []),
+      ...(label ? [{ url: `/apps/${app}/labels/${label}/`, name: label }] : []),
```

Comment:
Why do we need this?

Reply:
Leftover from an earlier experiment. Will take it out.